src/tools/reconcileLegacyReports.js

```diff
@@ -1,7 +1,7 @@
 import reconcileLegacyReports from '../services/legacyreports';
 import { auditLogger } from '../logger';
 
-reconcileLegacyReports().then(process.exit(0)).catch((e) => {
```

Comment:
I missed this on my review too - but this should have been `.then(() => process.exit(0))` if we wanted to keep it in. The script already exits with 0 if the `catch` callback isn't called though, so I figured the cleaner thing was to just remove it.

Reply:
The reason I put that in was without it it was hanging and taking a long time to exit. If it runs ok without it in cloud.gov, I am fine with it being gone though.